core/src/main/java/com/klaytn/caver/rpc/RPC.java

```diff
@@ -80,4 +82,8 @@ public Net getNet() {
         return net;
     }
 
+    @Override
```

Comment:
Would you add method description for java doc?

Reply:
@sirano11 Added comments :)